File: libvoikko/java/src/main/java/org/puimula/libvoikko/Sentence.java
package org.puimula.libvoikko;

public class Sentence {

    private final String text;
    private final SentenceStartType nextStartType;
    
    public Sentence(String text, SentenceStartType nextStartType) {
        this.text = text;
        this.nextStartType = nextStartType;
    }

    public String getText() {
        return text;
    }

    public SentenceStartType getNextStartType() {
        return nextStartType;
    }
}

File: libvoikko/cs/libvoikko/Sentence.cs
using System;
namespace libvoikko
{
	public class Sentence
	{
		private readonly string text;
		public string Text
		{
			get
			{
				return text;
			}
		}
		
		private readonly SentenceStartType nextStartType;
		public SentenceStartType NextStartType
		{
			get
			{
				return nextStartType;
			}
		}
		
		public Sentence(string text, SentenceStartType nextStartType)
		{
			this.text = text;
			this.nextStartType = nextStartType;
		}
	}
}

